doc/source/multivisit_products.rst

```diff
@@ -0,0 +1,243 @@
+SVM processing performed by the HST calibration pipeline and these aligned products then get used as inputs to generate
+the MVM mosaics.
+
+MVM mosaics include as many exposures as possible but due to these reasons, a given MVM mosaic can contain exposures
+fit to different astrometric catalogs.  Exposures which do not overlap other exposures in the MVM mosaic can only use
+the WCS defined during SVM processing and that may not be the most accurate GAIA-based catalog available.  This can result
+in errors in the relative alignment between those exposures and the rest of the exposures in the mosaic which have been
+aligned to the most accurate catalog available.  These errors may not be large (less than a few pixels in nearly all cases),
+but care must be taken to interpret the positions of sources across a SkyCell due to such effects.
+
+
+Effects of Proper Motion
+-------------------------
+HST has been taking images for over 30 years, with ACS being active since 2002 and WFC3 having been installed in 2009.
+As a result, there have been some fields which have been observed multiple times over those decades allowing HST with
+it's high resolution to observe the stellar proper motions.  Any source with proper motions of a few milli-arcseconds
+motion per year or more could potentially be observed moving when comparing images taken over the life of HST.  Such
+sources can be found in observations of sources like nearby globular clusters (e.g., 47Tuc or Omega Cen) or open clusters
+(like M35).
+
+Unfortunately, this makes aligning images of such high proper motion sources difficult to interpret.  Measurements of
+each sources proper motions have been included in the catalog information for sources in the GAIA-based astrometric
+catalogs for all catalogs derived using GAIADR2 or later.  The HST images taken during the same visit are then aligned
+to the GAIA-based catalog, using the GAIAeDR3 catalog by default through 2022 during the SVM processing.  This SVM
+alignment generates a consistent fit for the proper motions at a single epoch.  These SVM-aligned exposures are then
+used, without further alignment, to generate the final MVM mosaic of all exposures taken over the life of HST.  As a result,
+each epoch represents a single snapshot of the sources at one time.  The sources in the field with large proper motions,
+on the other hand, are not going to align as they have moved from one epoch.  This will result in those sources showing
+up with smeared PSFs or even double-exposures in the final MVM mosaic image while the background sources
+(typically background galaxies or globular clusters) will be well aligned.
+
+These errors can actually be a good way to spot high proper motion sources for later study.  In addition, improvements
+to the MVM processing code are being developed to allow for such fields to processed such that observations taken at
```

Comment:
Change "fields to processed" ==> "fields to be processed".

Reply:
Fixed in 175ee20.